frontend/javascripts/oxalis/model/sagas/update_actions.js

```diff
@@ -136,6 +136,10 @@ export type RevertToVersionUpdateAction = {|
     sourceVersion: number,
   |},
 |};
+export type RemoveFallbackLayerAction = {|
+  name: "removeFallbackLayer",
+  value: {},
```

Comment:
Can the value property be removed? Or is there some code which requires this type to look that way?

Reply:
The value property is necessary for the `AsServerAction<...>` conversion.